save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/DeleteOrganizationButton.kt

```diff
@@ -115,3 +106,13 @@ external interface DeleteOrganizationButtonProps : Props {
      */
     var classes: String
 }
+
+private enum class ModalPurpose {
+    DELETE_MODAL,
+    ERROR_MODAL,
+    ;
+
+    companion object {
```

Comment:
Not sure it is needed here, just in case

Reply:
Done